use_case_examples/tfhers_interoperability/AuthenticationWithDecisionTree.ipynb

```diff
@@ -0,0 +1,695 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Introduction:\n",
+    "\n",
+    "\n",
+    "Concrete ML (CML) now supports model compilation to the TFHE-rs format, enabling deeper interoperability between the CML and TFHE-rs frameworks. This allows developers to combine CML’s intuitive, Python-based workflow with the performance and low-level control of TFHE-rs for advanced privacy-preserving applications.\n",
```

Comment:
`Concrete ML (CML) now supports model compilation to the TFHE-rs format,` -> `Concrete ML supports TFHE-rs format inputs and outputs for FHE ML models`

don't use `CML`, put `Concrete ML` everywhere please

also remove `performance` for TFHE-rs